pkg/reconciler/pipeline/validate.go

```diff
@@ -133,6 +133,15 @@ func validateVertex(v dfv1.AbstractVertex) error {
 	if min > max {
 		return fmt.Errorf("vertex %q: max number of replicas should be greater than or equal to min", v.Name)
 	}
+	for _, ic := range v.InitContainers {
+		if ic.Name == dfv1.CtrInit ||
+			ic.Name == dfv1.CtrMain ||
+			ic.Name == "numa" ||
```

Comment:
since we do not have `numa` container, we can remove `ic.Name == "numa"` check, we can fix that by changing the const value of `CtrMain` later.

In case someone creates a container by the name `numa` by then, it won't be backward compatible 😄

Reply:
Could you change the value of `CtrMain` to `numa` in this PR?